docs/content/users/configuration/config.md

```diff
@@ -198,6 +198,14 @@ Whether [`ddev start`](../usage/commands.md#start) should be interrupted by a fa
 | -- | -- | --
 | :octicons-file-directory-16: project<br>:octicons-globe-16: global | `false` | Can be `true` or `false`.
 
+## `global_config_path`
```

Comment:
Interested to know why this was needed, and why it isn't the *directory* instead of the *file*. project_list.yaml should still be a peer of this in the same directory, true?

If it's controlled by XDG_CONFIG_HOME, why does it have its own config element?

Should this be mostly hidden (maybe it is, haven't gotten that far...)